tests/src/test/scala/cats/mtl/tests/IOTests.scala

```diff
@@ -18,13 +18,14 @@ package cats.mtl.tests
 
 import cats.effect.testkit.TestInstances
 import cats.mtl.effect.instances.io._
-import cats.mtl.laws.discipline.{HandleTests, StatefulTests}
 import cats.effect.IO
 import cats.effect.IOLocal
 import munit.CatsEffectSuite
 import munit.DisciplineSuite
 
 class IOTests extends CatsEffectSuite with DisciplineSuite with TestInstances {
+  import cats.mtl.laws.discipline.{HandleTests, StatefulTests}
+
```

Comment:
I don't understand why the imports need to be here, but the other tests do it too.